### ISSUE TITLE
Why Multiple Identities?

### ISSUE THREAD

**Issue by jsquire** [Azure/azure-functions-dotnet-worker#638]
# Issue Transfer

This issue has been transferred from the Azure SDK for .NET, [#24204](https://github.com/Azure/azure-sdk-for-net/issues/24204).

## Details

Under what circumstances could there be more than one identity?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ee033456-b610-5325-5e48-ecb12b821c9b
* Version Independent ID: 99068dbe-d073-7199-e957-ece9c70b8db9
* Content: [HttpRequestData.Identities Property (Microsoft.Azure.Functions.Worker.Http) - Azure for .NET Developers](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.functions.worker.http.httprequestdata.identities?view=azure-dotnet)
* Content Source: [xml/Microsoft.Azure.Functions.Worker.Http/HttpRequestData.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/master/xml/Microsoft.Azure.Functions.Worker.Http/HttpRequestData.xml)
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by liliankasem** [comment]
The ask from the functions team is not clear - @ rjarrattmsft please provide more information

**Comment by rjarrattmsft** [comment]
I raised this two years ago, so it is a bit of a surprise to be asked about this now. My question is really very simple, in the documentation linked to the Identities properties is an IEnumerable<ClaimsIdentity>. This suggests there could be more than one identity and all I wanted to understand is why there can be more than one. This matters when consuming code needs to process this property to determine the identity that is calling, how would the consuming code know which identity to check, why might it need to check all of the identities, is there one identity in particular that should be checked. Basically, under what circumstances could there be more than one identity in this property?

**Comment by liliankasem** [comment]
Yeah sorry about that; we're going through older issues and trying to get this repos issues back into a good, up-to-date state. 

>  The documentation linked to the Identities properties is an IEnumerable

Can you link to the documentation you're referencing? I'm not really sure what identity you're talking about

cc: @mattchenderson who's more familiar with managed identity topics.

**Comment by rjarrattmsft** [comment]
The link is at the top of this issue it is https://learn.microsoft.com/en-us/dotnet/api/microsoft.azure.functions.worker.http.httprequestdata.identities?view=azure-dotnet

**Comment by rjarrattmsft** [comment]
I don't know why it has been marked as stale, I have responded with the required information and not been asked anything else.

**Comment by liliankasem** [comment]
> The link is at the top of this issue it is https://learn.microsoft.com/en-us/dotnet/api/microsoft.azure.functions.worker.http.httprequestdata.identities?view=azure-dotnet

Ah sorry I missed that in the issue template. @brettsam - do you know more about the ClaimsIdentity IEnumerable?

**Comment by rjarrattmsft** [comment]
Well really that is the point of my question. It is the identity of the caller, so my question is: under what circumstances would there be more than one identity?
